deploy/L2TokenBridgeSpell.sol

```diff
@@ -44,6 +47,7 @@ contract L2TokenBridgeSpell {
         l2Bridge = L2TokenBridgeLike(l2Bridge_);
     }
 
+    function upgradeToAndCall(address newImp, bytes memory data) external { l2Bridge.upgradeToAndCall(newImp, data); }
```

Comment:
Note that I chose to ignore that this function is normally `payable` to keep things simpler, as it doesn't seem useful to be able to pass value to the initialization function (and the `L2GovernanceRelay` doesn't explicitly supports receiving ether, though there is still the `selfdestruct` workaround).